py-polars/polars/internals/dataframe/groupby.py

```diff
@@ -206,7 +207,7 @@ def apply(self, f: Callable[[pli.DataFrame], pli.DataFrame]) -> DF:
 
         return self._dataframe_class._from_pydf(self._df.groupby_apply(by, f))
 
-    def agg(self, aggs: pli.Expr | Sequence[pli.Expr]) -> pli.DataFrame:
+    def agg(self, aggs: IntoExpr | Sequence[IntoExpr]) -> pli.DataFrame:
```

Comment:
As you noted, we accept iterables too. So in accordance with your statement to "\prefer prioritizing correctness in type annotations and recommending typical use cases in examples.", we should use `Iterable` here.

```suggestion
    def agg(self, aggs: IntoExpr | Iterable[IntoExpr]) -> pli.DataFrame:
```

Reply:
Thanks, I first tried `IntoExpr | Sequence` and changed it to `Iterable`, so this left unchanged from the previous trial.